app/src/main/res/layout/unanswered_survey_view.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
+  style="@style/ActivityFeedCard"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:card_view="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginBottom="@dimen/grid_new_2"
+  android:focusable="true"
+  card_view:cardCornerRadius="0dp"
+  card_view:cardUseCompatPadding="true">
+
+  <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+      android:id="@+id/survey_avatar_image"
+      android:layout_width="@dimen/discovery_activity_photo_width"
+      android:layout_height="@dimen/discovery_activity_photo_height"
+      android:layout_alignParentStart="true"
+      android:layout_margin="@dimen/grid_new_2"
+      tools:ignore="ContentDescription" />
+
+    <TextView
+      android:id="@+id/survey_title"
+      style="@style/SubheadPrimary"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_alignTop="@id/survey_avatar_image"
+      android:layout_marginBottom="@dimen/activity_padding_x"
+      android:layout_toEndOf="@id/survey_avatar_image"
+      android:gravity="center_vertical"
+      android:paddingTop="@dimen/activity_padding_x"
+      android:text="@string/Survey"
+      android:textAlignment="gravity" />
+
+    <TextView
+      android:id="@+id/survey_text"
+      style="@style/Headline"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_below="@id/survey_avatar_image"
+      android:layout_margin="@dimen/grid_new_2" />
+
+    <View
+      android:id="@+id/separator"
```

Comment:
we actually do have a `horizontal_line_1dp_view` you can reuse here with a `<layout/>` tag